packages/react-native-win32/src/Libraries/Network/RCTNetworking.win32.ts

```diff
@@ -1,11 +1,101 @@
 /**
- * @providesModule RCTNetworking
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
  */
 'use strict';
 
 // Do not require the native RCTNetworking module directly! Use this wrapper module instead.
 // It will add the necessary requestId, so that you don't have to generate it yourself.
 
-import { RCTNetworking } from './Network';
```

Comment:
Network isn't directly included anywhere. Moved thw RCTNetworking code back to the main file for consistency with other platforms.